pages/news/2020-05-22-why-mixxx-will-not-support-streaming.html

```diff
@@ -0,0 +1,84 @@
+Free software is not about laws; it is about freedom.
+Even if you do not use the recording or broadcasting features of Mixxx, or you personally would agree to the restrictive terms of a streaming service, we want to maintain your trust that
+**Mixxx will always work for you, the user. Mixxx will never work for a company against your wishes.**
+
+Some users and Mixxx developers argue that a user should be able to agree to the restrictive terms of a streaming company if they want.
+You can still do that with other software, but that does not mean we have to support it in Mixxx.
+Mixxx does not have to do everything. It does not mix videos, nor control lights, nor does it brew coffee.
+Mixxx has always been a DJ application for mixing music files on your computer and that is what it will continue to be.
+
+Besides the restrictive terms of streaming services, there are many more reasons to not support streaming in Mixxx.
+
+Because everyone has the freedom to change how Mixxx works, it would be easy to disable restrictions on recording and broadcasting from streaming services.
+If a user user didn't want to do this on their own, it would be easy for anyone  to do it for them and host downloads of modified versions of Mixxx that could record and broadcast from streaming services.
+It would be difficult for us to distance these unofficial modifications from Mixxx, which would reflect badly on Mixxx.
+Other software teams [already](https://kodi.tv/article/piracy-box-sellers-and-youtube-promoters-are-killing-kodi) [have](https://kodi.tv/article/kodi-builds-streaming-add-ons-and-free-lunches) [trouble](https://kodi.tv/article/is-kodi-legal) [distancing their name](https://kodi.tv/article/in-defence-of-our-good-name) from illegal third party modifications/addons.
+Without supporting streaming, Mixxx already has [enough trouble](/news/2020-05-22-you-dont-need-to-pay-for-mixxx/) with third parties selling copies of Mixxx without complying with the license or trademark or providing support.
+We do not want to make this problem worse.
+Mixxx is made by a small team of volunteers (we are always looking for new people to [join us](/get-involved)!).
+We want to spend our time making Mixxx better, not defending the reputation of our work from unscrupulous third parties.
+
+There are many practical reasons to not support streaming services as well.
+Supporting streaming services in Mixxx would require a lot of work to implement, perhaps months of work.
+As with all Mixxx features, this would be done entirely by volunteers with limited time.
+Those volunteers would not be using that time working on other features.
+
+All this hard work could be wasted at the whim of the streaming company or a record company they get music from. 
+Either the streaming company or record company could decide to no longer allow Mixxx to use the streaming service or they could go out of business.
```

Comment:
```suggestion
All this hard work could be in vain if the streaming company or a record company they get music from would suddenly revoke access for Mixxx or go out of business.
```

Reply:
I prefer the current wording. I think combining these into one sentence is putting too much into one sentence.

Reply:
That's a point, but I feel like right now there is too much duplication between them.